tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Split into `build.json` so that `spec` and `src` folders are not included in the `dist` folder. We cannot use:

```
"include": [
    "./src"
  ]
```

because tests then don't have access to `Conviva.d.ts` types. If we remove `include` completely, then redundant files are built into the `dist` folder.